package/boot/uboot-sunxi/Makefile

```diff
@@ -349,7 +358,7 @@ UBOOT_TARGETS := \
 UBOOT_CONFIGURE_VARS += USE_PRIVATE_LIBGCC=yes
 
 UBOOT_MAKE_FLAGS += \
-	BL31=$(STAGING_DIR_IMAGE)/bl31.bin
+	BL31=$(STAGING_DIR_IMAGE)/bl31_$(ATF).bin
```

Comment:
The ```bl31_$(ATF).bin``` is only created in a later patch, it would bet better to have this compiling all the time.

Reply:
The ATF variable is created in the very first patch, and the UBOOT_MAKE_FLAGS is updated accordingly in the same one.